src/zyre_selftest.c

```diff
@@ -31,10 +31,6 @@ static test_item_t
 all_tests [] = {
     { "zyre", zyre_test },
     { "zyre_event", zyre_event_test },
-    { "zre_msg", zre_msg_test },
```

Comment:
NOTE:

These need to be added back in! They are private symbols so they can't be referenced from the external zyre_selftest binary.

But since they are private classes used by public ones, the simplest and clearest solution is to call the *_test method from the public class that uses the private one. Like I did in CMZQ:

https://github.com/zeromq/czmq/pull/1549/files